.github/workflows/deploy_whole_doc.yml

```diff
@@ -126,7 +146,7 @@ jobs:
   invalidate_cloudfront:
     name: Invalidate CloudFront
     needs: [deploy_child_repo]
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-20.04
```

Comment:
Same here

Reply:
latest is not really a proper choice because we never now if it will work when the latest ubuntu change on GitHub, it is a better thing to fix the ubuntu version